File: php-laravel/lib/app/Models/RequestTimeFilterPostcodeSectorsDepartureSearch.php
<?php
/**
 * RequestTimeFilterPostcodeSectorsDepartureSearch
 */
namespace app\Models;

/**
 * RequestTimeFilterPostcodeSectorsDepartureSearch
 */
class RequestTimeFilterPostcodeSectorsDepartureSearch {

    /** @var string $id */
    private $id;

    /** @var \app\Models\RequestTransportation $transportation */
    private $transportation;

    /** @var int $travelTime */
    private $travelTime;

    /** @var \DateTime $departureTime */
    private $departureTime;

    /** @var double $reachablePostcodesThreshold */
    private $reachablePostcodesThreshold;

    /** @var \app\Models\RequestTimeFilterPostcodeSectorsProperty[] $properties */
    private $properties;

    /** @var \app\Models\RequestRangeFull $range */
    private $range;

}

File: fsharp-giraffe-server/traveltimeplatform/src/model/RequestTimeFilterPostcodeSectorsDepartureSearch.fs
namespace traveltimeplatform.Model

open System
open System.Collections.Generic
open traveltimeplatform.Model.RequestRangeFull
open traveltimeplatform.Model.RequestTimeFilterPostcodeSectorsProperty
open traveltimeplatform.Model.RequestTransportation

module RequestTimeFilterPostcodeSectorsDepartureSearch = 

  //#region RequestTimeFilterPostcodeSectorsDepartureSearch


  type RequestTimeFilterPostcodeSectorsDepartureSearch = {
    Id : string;
    Transportation : RequestTransportation;
    TravelTime : int;
    DepartureTime : DateTime;
    ReachablePostcodesThreshold : double;
    Properties : RequestTimeFilterPostcodeSectorsProperty[];
    Range : RequestRangeFull;
  }
  //#endregion
  